src/components/StoriesGroup/components/StoriesPreview/StoriesPreview.scss

```diff
@@ -0,0 +1,71 @@
+@use '../../../variables';
+
+$block: '.#{variables.$ns}stories-group-preview';
+
+#{$block} {
+    user-select: none;
+    margin-bottom: 16px;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 8px;
+
+    &__slider-previe-list-wrapper {
+        width: 568px; // widht of 12 preview items
```

Comment:
<img width="957" alt="image" src="https://user-images.githubusercontent.com/24455680/205589120-4b88d750-8e77-437e-b0aa-6ced6a5bee0f.png">
<img width="946" alt="image" src="https://user-images.githubusercontent.com/24455680/205589152-75be5ab8-ec12-465b-b7ed-811f479ee700.png">

Shouldn't we keep in in the middle? I would prefer do not hardcode it here